tools/events-automation/README.md

```diff
@@ -1 +1,71 @@
-TODO: write this
\ No newline at end of file
+### Event Class:
```

Comment:
If there are any call-outs or specific questions to the reviewers for a particular line(s) of code, you can place comments indicating so.